holidays/countries/angola.py

```diff
@@ -9,10 +9,9 @@
 #  Website: https://github.com/dr-prodigy/python-holidays
 #  License: MIT (see LICENSE file)
 
-from datetime import date
+from datetime import date, timedelta
```

Comment:
Maybe import `timedelta as td`? Then lines length will be same.

Reply:
Actually, it was [my original intent](https://github.com/dr-prodigy/python-holidays/issues/891) to do it that way. I'm not sure why I didn't stick to the plan. Thank for pointing out, @KJhellico!

It's great to have you back! It felt somewhat empty without your reviews here. I really appreciate them!